src/styles/components/v-tooltip.css

```diff
@@ -0,0 +1,33 @@
+.ec-popover {
+  @apply tw-block;
+
+  /* base z-indexing for popover and related modifiers */
+  @apply tw-z-level-1;
```

Comment:
This is not the same as `@include z-index-all-levels` in scss. That mixin generates this:
```
@mixin z-index-all-levels {
  @include z-index-level-1;

  &--level-2 {
    @include z-index-level-2;
  }

  &--level-3 {
    @include z-index-level-3;
  }

  &--modal {
    @include z-index-modal;
  }

  &--tooltip {
    @include z-index-tooltip;
  }

  &--notification {
    @include z-index-notification;
  }
}
```

Reply:
Thank you Milos for spotting it! I've added the migrated mixin as well and added it to the file